# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ScanActionGoal action_goal
ScanActionResult action_result
ScanActionFeedback action_feedback
